src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx

```diff
@@ -0,0 +1,206 @@
+      meta
+    }
+  };
+}
+
+Amplify Auth can be configured to use a existing resources. If you are in a team setting or part of a company that has previously created auth resources, you have a few different options to configure your application to use existing auth resources:
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+- [Add auth configuration as an output of the backend](#add-auth-as-a-backend-output)
+- [Configure the client library directly (without an Amplify backend)](#configure-the-client-library-directly)
+
+</InlineFilter>
+{/* merge content after hosting outputs schema */}
+<InlineFilter filters={["android", "flutter", "swift"]}>
+
+- [Add auth configuration as an output of the backend](#add-auth-as-a-backend-output)
+
+</InlineFilter>
+
+<Callout info>
+
+**Note:** when using existing auth resources, it may be necessary to add policies or permissions for your authenticated and unauthenticated IAM roles. These changes must be performed manually using the [AWS Cloud Development Kit (AWS CDK)](https://aws.amazon.com/cdk/)
+
+</Callout>
+
+## Add auth as a backend output
+
+The easiest way to get started with your existing resource is to use `backend.addOutput` to surface auth configuration to `amplify_outputs.json` automatically. In it's simplest form:
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend"
+
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
+ */
+const backend = defineBackend({})
+
+backend.addOutput({
+  auth: {
+    aws_region: "<your-cognito-aws-region>",
+    user_pool_id: "<your-cognito-user-pool-id>",
+    user_pool_client_id: "<your-cognito-user-pool-client-id>",
+    identity_pool_id: "<your-cognito-identity-pool-id>",
+    username_attributes: ["email"],
+    standard_required_attributes: ["email"],
+    user_verification_types: ["email"],
+    unauthenticated_identities_enabled: true,
+    password_policy: {
+      min_length: 8,
+      require_lowercase: true,
+      require_uppercase: true,
+      require_numbers: true,
+      require_symbols: true,
+    }
+  }
+})
+```
+
+<Callout warning>
+
+**Warning:** if you are creating an auth resource with `defineAuth`, you cannot override the default auth configuration automatically surfaced to `amplify_outputs.json` by Amplify.
+
+</Callout>
+
+You can also use the CDK to dynamically reference existing resources, and use metadata from that resource to set up IAM policies for other resources, or reference as an authorizer for a custom REST API:
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend"
+import { UserPool, UserPoolClient } from "aws-cdk-lib/aws-cognito"
+
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
+ */
+const backend = defineBackend({})
+
+const authStack = backend.createStack("ExistingAuth")
```

Comment:
Does this approach line up with the Storage approach? What about customers that don't want a "dummy stack" to add these outputs?

Reply:
that is in the previous example